src/modules/taskClass.js

```diff
@@ -0,0 +1,8 @@
+export default class Task {
+  constructor(arr, description) {
+    this.description = description;
+    this.complete = false;
+    this.index = arr.length;
```

Comment:
- [ ] The property index set to the value of the new array length (i.e. if a 4th task is added to the list, the index of that task should equal to 4) which means the index should start by 1 and not 0. Please kindly fix that 🙃 😉 

<img width="485" alt="Screen Shot 2023-01-20 at 18 41 42" src="https://user-images.githubusercontent.com/74501523/213741629-2224cfa2-50d3-40aa-9cff-5534fc57258c.png">

Reply:
I got you @cynthiainga

Reply:
- Great job updating the index 👍🏽 

<img width="1221" alt="Screenshot 2023-01-20 at 17 28 23" src="https://user-images.githubusercontent.com/58692590/213752090-a8ada10c-432c-4d19-82c9-4b10f4fed150.png">

Reply:
Thank you 🙏 @Whoistolu